dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -120,6 +173,38 @@ class EvaluateRubricJob < ApplicationJob
     # We gracefully just fail, here, and we do not file this exception
   end
 
+  RETRIES_ON_RATE_LIMIT = 3
+
+  # Retry on any reported rate limit (429 status) 'exponentially_longer' waits 3s, 18s, and then 83s.
+  retry_on TooManyRequestsError, wait: :exponentially_longer, attempts: RETRIES_ON_RATE_LIMIT do |job, error|
+    # Job arguments are always serializable, so we just pull out the hash
+    # and send it as context.
+    options = job.arguments.first
+
+    # Send it to honeybadger even though we are handling the error via retry
+    Honeybadger.notify(
+      error,
+      error_message: "Retrying rubric evaluation job due to rate limiting (429).",
+      context: options
+    )
+  end
```

Comment:
what do you think about including more details from the openai 429 response here? or will that already be included in the response object that is embedded in the `error`? the openai response headers (whether 429 or not) are supposed to tell you interesting things about your current usage with respect to the rate limits: https://platform.openai.com/docs/guides/rate-limits/rate-limits-in-headers

I think this is nice-to-have for now if you see an easy way, otherwise I'm fine with delaying until we actually start having problems with this where we think this info will be useful.